core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -1094,7 +1095,7 @@ mod tests {
                     &Keypair::new(),
                     &Pubkey::new_unique(),
                     1,
-                    i * 10,
+                    i * 10_000,
```

Comment:
why is this necessary?

Reply:
the test requires 4 test transactions to line up in queue in order of [0, 1, 2, 3] based on priority. Before feature `reserve_minimal_cus_for_builtin_instructions` is enabled, tx cost is `system + compute-budget = 300 CUs`; with feature enabled, tx cost is `2 * 3K`; then cu price of `10` is too small to guarantee 4 test txs always in 0 to 3 order, made this test flacky. Safer to increase prio fee to the level other tests are.